Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -213,7 +213,10 @@ final class WorkspaceTests: XCTestCase {
                     """
                 )
 
-                XCTAssertMatch(ws.interpreterFlags(for: foo), [.equal("-swift-version"), .equal("6")])
+                XCTAssertMatch(
+                    ws.interpreterFlags(for: foo),
+                    (SwiftVersion.current.isDevelopment) ? [.equal("-swift-version"), .equal("6")] : []
```

Comment:
question: instead of setting the expected value based on `SwiftVersion.current.isDevelopment`, should we instead modify line 208 to use a specific `// swift-tools-version:5.10`, or whichever version it's supposed to be?